cfn-resources/project-invitation/cmd/resource/model.go

```diff
@@ -0,0 +1,24 @@
+// Code generated by 'cfn generate', changes will be undone by the next invocation. DO NOT EDIT.
+// Updates to this type are made my editing the schema file and executing the 'generate' command.
+package resource
+
+// Model is autogenerated from the json schema
+type Model struct {
+	ApiKeys         *ApiKeyDefinition `json:",omitempty"`
+	CreatedAt       *string           `json:",omitempty"`
+	ExpiresAt       *string           `json:",omitempty"`
+	ProjectId       *string           `json:",omitempty"`
+	Id              *string           `json:",omitempty"`
+	TotalCount      *float64          `json:",omitempty"`
+	PageNum         *int              `json:",omitempty"`
```

Comment:
total count and page num is used for the listing, but i dont see we are using it, or should we use it to make an efficient listing?